modules/docextract/lib/refextract_kbs.py

```diff
@@ -194,7 +193,6 @@ def institute_num_pattern_to_regex(pattern):
     """
     simple_replacements = [
                             ('9',    r'\d'),
-                            ('9+',   r'\d+'),
```

Comment:
in fact doesn't look like there is a '9+' left in the kb -- is this why the pattern is removed here?